plugins/modules/ipavault.py

```diff
@@ -555,6 +602,10 @@ def main():
                         commands.append([name, 'vault_remove_owner',
                                          owner_del_args])
 
+                    if vault_type == 'symmetric' \
+                       and 'ipavaultsalt' not in args:
+                        args['ipavaultsalt'] = os.urandom(32)
```

Comment:
is it ok generate a random salt for symmetric? Are you able to retrieve this information afterwards without this salt?

Reply:
Yes you can retrieve the `salt` (in the CLI, but not with Ansible). It is the same way the CLI command does this.